<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RandomColorClass</class>
 <widget class="QMainWindow" name="RandomColorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RandomColor</string>
  </property>
  <widget class="QWidget" name="m_pCentralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="m_pLblA">
      <property name="text">
       <string>A</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="m_pLblB">
      <property name="text">
       <string>B</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="m_pLblC">
      <property name="text">
       <string>C</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="m_pLblD">
      <property name="text">
       <string>D</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="m_pLblE">
      <property name="text">
       <string>E</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="m_pLblF">
      <property name="text">
       <string>F</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="m_pLblG">
      <property name="text">
       <string>G</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="m_pLblH">
      <property name="text">
       <string>H</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="m_pLblI">
      <property name="text">
       <string>I</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="m_pLblJ">
      <property name="text">
       <string>J</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="m_pLblK">
      <property name="text">
       <string>K</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="m_pLblL">
      <property name="text">
       <string>L</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="m_pLblM">
      <property name="text">
       <string>M</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="m_pLblN">
      <property name="text">
       <string>N</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="m_pLblO">
      <property name="text">
       <string>O</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="m_pLblP">
      <property name="text">
       <string>P</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="m_pLblQ">
      <property name="text">
       <string>Q</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="m_pLblR">
      <property name="text">
       <string>R</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="m_pLblS">
      <property name="text">
       <string>S</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="m_pLblT">
      <property name="text">
       <string>T</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_pMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>277</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGenerate_Color">
    <property name="title">
     <string>Generate Color</string>
    </property>
    <addaction name="m_pActionTest"/>
    <addaction name="m_pActionKellyColors"/>
   </widget>
   <addaction name="menuGenerate_Color"/>
  </widget>
  <widget class="QToolBar" name="m_pMainToolBar">
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_pActionTest"/>
   <addaction name="m_pActionKellyColors"/>
  </widget>
  <widget class="QStatusBar" name="m_pStatusBar"/>
  <action name="m_pActionTest">
   <property name="icon">
    <iconset resource="randomcolor.qrc">
     <normaloff>:/RandomColor/Icon/random.png</normaloff>:/RandomColor/Icon/random.png</iconset>
   </property>
   <property name="text">
    <string>Random</string>
   </property>
  </action>
  <action name="m_pActionKellyColors">
   <property name="icon">
    <iconset resource="randomcolor.qrc">
     <normaloff>:/RandomColor/Icon/colors.png</normaloff>:/RandomColor/Icon/colors.png</iconset>
   </property>
   <property name="text">
    <string>KellyColors</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="randomcolor.qrc"/>
 </resources>
 <connections/>
</ui>
